server/__init__.py

```diff
@@ -134,7 +140,7 @@ def register_routes_common(app):
   app.register_blueprint(browser_api.bp)
   app.register_blueprint(choropleth.bp)
   app.register_blueprint(csv.bp)
-  app.register_blueprint(disaster_dashboard.bp)
+  app.register_blueprint(disaster_api.bp)
```

Comment:
This should move to the "stanford" instances too (and we would need to move the data json to the right gcp instance)

Reply:
Done!

Reply:
I think this move may be causing tests for the disaster apis to fail because the blueprint isn't registered for the environment in which the tests are being run anymore

Reply:
ah good catch. we can also install it if `cfg.TEST`

Reply:
Done!